students/rmart300/lesson09/donor_collection.py

```diff
@@ -0,0 +1,92 @@
+import os
+from donor import Donor
+
+class DonorCollection(object):
+
+    letter_directory = 'temp/'
+
+    def __init__(self):
+        # dictionary to hold donors and list of donation amounts
+        donor_list = ['Fred Smith', 'Terrie Ann', 'Murray Martin', 'Josh Jones', 'Jane Doe']
+        self.donation_dict = {}
+        self.donor_set = set()
+
+        # build initial dictionary of donors and donation amounts
+        amount_list = [ 500, 100, 1000]
```

Comment:
Donations should probably be stored as an attribute on the `Donor` object, since they belong to a particular donor